SSH page (#7368)
circleci/circleci-docs: jekyll/_cci2/ssh-access-jobs.md

Often the best way to troubleshoot problems is to SSH into a job and inspect things like log files, running processes, and directory paths. CircleCI gives you the option to access all jobs via SSH. Read our [blog post](https://circleci.com/blog/debugging-ci-cd-pipelines-with-ssh-access/) on debugging CI/CD pipelines with SSH.

When you log in with SSH, you are running an interactive login shell. You may be running the command on top of the directory where the command failed the first time, or you may be running the command from the directory one level up from where the command failed (e.g. `~/project/` or `~/`). Either way, you will not be initiating a clean run. You may wish to execute `pwd` or `ls` to ensure that you are in the correct directory.

> Please note that a default CircleCI pipeline executes steps in a non-interactive shell and there is the possibility that running steps using an interactive login may succeed, while failing in non-interactive mode.

Reviewer: ```suggestion
Please note that a default CircleCI pipeline executes steps in a non-interactive shell. There is a possibility that running steps using an interactive login may succeed, but in non-interactive mode.
```
I think something like this makes it more readable

If you are using the Windows executor you will need to pass in the shell you want to use when using SSH. For example, To run  `powershell` in your build you
would run: `ssh -p <remote_ip> -- powershell.exe`. Consider reading the [Hello world on Windows]({{site.baseurl}}/hello-world-windows) page to learn more.

The build virtual machine (VM) will remain available for an SSH connection for **10 minutes after the build finishes running** and then automatically shut down (or you can cancel it). After you SSH into the build, the connection will remain open for **one hour** for customers on the Free plan, or **two hours** for all other customers.

If your job has parallel steps, CircleCI launches more than one VM to perform them. You will see more than one 'Enable SSH' and 'Wait for SSH' section in the build output.

## Debugging: "permission denied (publickey)"
{: #debugging-permission-denied-publickey }

If you run into permission troubles trying to SSH to your job, try the following in the sections below.

### Ensure authentication with GitHub/Bitbucket
{: #ensure-authentication-with-githubbitbucket }

> A single command can be used to test that your keys are set up as expected. For GitHub run:

Reviewer: ```suggestion
A single command can be used to test that your keys are set up as expected. For GitHub, run:
```

> For GitHub run

Reviewer: Do we mean to remove this line?

Reviewer: Line 56, that is
